x-pack/test/security_solution_cypress/cypress/e2e/detection_response/detection_engine/rule_creation/event_correlation_rule_suppression_sequence.cy.ts

```diff
@@ -0,0 +1,52 @@
+/*
```

Comment:
Nit: I think test cases in `event_correlation_rule_suppression_sequence.cy.ts` and `event_correlation_rule_suppression.cy.ts` can be combined in one file - less maintanence

Reply:
In the course of reviewing the tests I came to the same conclusion 👍